### PR TITLE
fix: add developer documentation style guide

### PR DESCRIPTION
**Pull request by ffe9f8**

### CODE REVIEW

**Comment by ffe9f8** [comment]
A few questions:

1. Is this easy to use as a reference document? Since each section is short, the heading sizes may look very similar and the reader could easily get lost.
2. Thoughts on using tables vs not? There are no tables for the "Code comments" and "Lists" section because they were too large.
3. Should there be an example for how to use section headings properly?
4. Thoughts on this living solely on Github (like `CONTRIBUTING.md`) vs on the Axelar docs site? I personally think we could create a new section on the docs site with `CONTRIBUTING.md` and `STYLE-GUIDE.md`.

File: STYLE-GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+* Do not use the word “I.” When speaking on Axelar’s behalf, use “we.”
+
+|  **Recommended**  |  **Not recommended** |
+| --- | --- |
+|  Hardhat and Foundry are local development platforms. Use Remix if you prefer an online editor.  |  Hardhat and Foundry are local development platforms. I’d use Remix if I preferred an online editor.  |
+|  We recommend using Metamask for this tutorial.  |  I recommend using Metamask for this tutorial.  |
+
+### When referring to a third party user or developer
+
+* Use the singular “they” when referring to a third party user or developer.
```
Comment (StephenFluin) [inline STYLE-GUIDE.md:30]: Swap out fancy quotes for standard double quotes "

File: STYLE-GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+**Recommended**
+
+`sendToMany` will need to do the following:
+
+* Get the address of a token from its symbol
+* Send funds to another address
+* Approve the gateway to spend funds
+* Encode the recipient addresses on the destination chain to ensure that they are `byte`s, since GMP messages must be of this type
+
+```
```
Comment (StephenFluin) [inline STYLE-GUIDE.md:65]: add language of this code like 

"```solidity`"

File: STYLE-GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+    IERC20(tokenAddress).approve(address(gateway), _amount);
+
+    bytes memory recipientAddressesEncoded = abi.encode(_destinationAddrs);
+}
+```
+
+### Section headings
+
+* Don’t skip heading types. A subheading under a title should be an H1, the nested subheading under that one an H2, and so forth.
```
Comment (StephenFluin) [inline STYLE-GUIDE.md:128]: Standard single quote instead of fancy single quote

File: STYLE-GUIDE.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+* `IERC20` – The ERC-20 token interface to access ERC-20-related functionality
+
+**Not recommended**
+
+Implement the following to send an interchain transaction with Axelar:
+
+* `AxelarExecutable` – This is a contract that handle a message on the destination chain once a transaction has been sent to the Axelar network.
+* `IAxelarGateway` – The Axelar Gateway
+* `IAxelarGasService` – The Axelar Gas Service
+* `IERC20` – This represents the ERC-20 token interface to access ERC-20-related functionality.
```
Comment (StephenFluin) [inline STYLE-GUIDE.md:202]: Can you add a section about capitalization? I've heard we don't capitalize "Axelar network" and other standards from Galen.

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: STYLE-GUIDE.md (hunk 4 above)
Comment (ffe9f8) [reply STYLE-GUIDE.md:202]: There is [another set of content guidelines for marketing and partner comms](https://www.notion.so/bright-ambert-2bd/Axelar-comms-content-rules-0c003b259d274e839f77f92034fc127b#e283efd1a0634d6ea1688f528aa93423) written by Galen, some of which contradict the guidelines set here. I'll meet with him to discuss standardizing to one set across all comms.

File: STYLE-GUIDE.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+**Recommended**
+
+`sendToMany` will need to do the following:
+
+* Get the address of a token from its symbol
+* Send funds to another address
+* Approve the gateway to spend funds
+* Encode the recipient addresses on the destination chain to ensure that they are `byte`s, since GMP messages must be of this type
+
+```solidity`
```
Comment (ffe9f8) [inline STYLE-GUIDE.md:65]: @StephenFluin I think the "```solidity`" formatting isn't right here

**Review by Olanetsoft — APPROVED** [review]

File: STYLE-GUIDE.md (hunk 4 above)
Comment (StephenFluin) [reply STYLE-GUIDE.md:202]: Let's add that in a second PR so we can get this one merged.

File: STYLE-GUIDE.md (hunk 5 above)
Comment (StephenFluin) [inline STYLE-GUIDE.md:65]:
```suggestion
```solidity
```

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**